### PR TITLE
Bo 1223

### PR DESCRIPTION
**Pull request by JustynaWr**
Please review the changes

### CODE REVIEW

File: first_project/first_app/models.py [hunk 1]
```diff
@@ -15,10 +16,10 @@ class Webpage(models.Model):
 
     def __str__(self):
         return self.name
-
+#nowe rzeczy
```
Comment (SuperXtra) [inline first_project/first_app/models.py:19]: Please change it!!!!

**Review by SuperXtra — COMMENTED** [review]
eqweqwe

File: first_project/first_app/models.py (hunk 1 above)
Comment (JustynaWr) [reply first_project/first_app/models.py:19]: can't be changed

**Comment by JustynaWr** [comment]
check again

**Review by SuperXtra — COMMENTED** [review]
Nope!

**Pull request merged**